opentelemetry/chart/templates/logs-collector.yaml

```
@@ -3,6 +3,7 @@ SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse c
 SPDX-License-Identifier: Apache-2.0
 */}}
 
+{{- if index .Values "open_telemetry" "LogsCollector" "enabled" }}
```

I'm not sure why you are using the `index` function here. To align with [best practices](https://helm.sh/docs/chart_best_practices/values/#naming-conventions), this should be camelCase. Like so
```suggestion
{{- if .Values.openTelemetry.LogsCollector.enabled }}
```

The renaming should then match everywhere.